tests/fireflyClient.test.ts

```diff
@@ -21,6 +21,7 @@ import {
   GetAccountDataResponse,
 } from "../index";
 import Web3 from "web3";
+import BigNumber from "bignumber.js"
```

Comment:
would be best to import big number every where from `firefly-exchange/library` This will save us from issues later down the road which can come when bignumber.js file is updated in firefly-client but not in library.